package/MDAnalysis/transformations/fit.py

```diff
@@ -82,38 +81,42 @@ def fit_translation(ag, reference, plane=None, weights=None):
     -------
     MDAnalysis.coordinates.base.Timestep
     """
-
-    if plane is not None:
-        axes = {'yz' : 0, 'xz' : 1, 'xy' : 2}
+    def __init__(self, ag, reference, plane=None, weights=None):
+        self.ag = ag
+        self.reference = reference
+        self.plane = plane
+        self.weights = weights
+
+
+    def __call__(self, ts):
+        if self.plane is not None:
```

Comment:
to be equivalent, `__call__` should only have the contents of `wrapped`, all other logic should go into `__init__`

Reply:
Makes sense!